redash/controllers.py

```diff
@@ -450,7 +450,7 @@ def delete(self, visualization_id):
 class QueryResultListAPI(BaseResource):
     @require_permission('execute_query')
     def post(self):
-        params = request.json
```

Comment:
I tried to keep it, but the test case seems to serialize it into a format that can't be parsed by `request.json`:

https://github.com/EverythingMe/redash/blob/master/tests/test_controllers.py#L33

Is it missing a header of some sort? I didn't have time to research. I just changed it to be consistent with the rest of the code in the controllers.